datalad/support/annexrepo.py

```diff
@@ -548,15 +548,9 @@ def get_tracking_branch(self, branch=None, remote_only=False,
     def _check_git_annex_version(cls):
         ver = external_versions['cmd:annex']
         # in case it is missing
-        msg = "Visit http://git-annex.branchable.com/install/"
-        # we might be able to do better
-        try:
-            linux_distribution_name = get_linux_distribution()[0]
-            if linux_distribution_name in {'debian', 'ubuntu'}:
-                msg = "Install  git-annex-standalone  from NeuroDebian " \
-                      "(http://neuro.debian.net)"
-        except:  # pragma: no cover
-            pass
+        msg = "Visit http://git-annex.branchable.com/install/ or " \
+              "install  git-annex-standalone  from NeuroDebian " \
+              "(http://neuro.debian.net) on Debian or Ubuntu"
```

Comment:
I like going with a static message, but if we do decide we want something tailored, we could delay the message construction until after the version check, paying the price only when we'd raise anyway.

Reply:
What about pointing to datalad-installer and handbook?  I would even start with handbook alone for now as a safer option:  "Visit http://handbook.datalad.org/en/latest/intro/installation.html for instructions on how to install DataLad and git-annex" (probably some shorter permalink?)